packages/plugin-core/src/components/lookup/LookupProviderV3.ts

```diff
@@ -1,41 +1,6 @@
-import {
-  ConfigUtils,
-  DNodeUtils,
-  FuseEngine,
-  LookupEvents,
-  NoteLookupUtils,
-  NoteProps,
-  NoteQuickInput,
-  NoteUtils,
-  SchemaModuleProps,
-  SchemaUtils,
-  VSCodeEvents,
-} from "@dendronhq/common-all";
-import { getDurationMilliseconds } from "@dendronhq/common-server";
-import { HistoryService } from "@dendronhq/engine-server";
+import { FuseEngine, NoteProps } from "@dendronhq/common-all";
 import _ from "lodash";
 import stringSimilarity from "string-similarity";
-import { CancellationTokenSource, window } from "vscode";
-import { NoteLookupCommand } from "../../commands/NoteLookupCommand";
-import { IDendronExtension } from "../../dendronExtensionInterface";
-import { Logger } from "../../logger";
-import { AnalyticsUtils } from "../../utils/analytics";
-import { NotePickerUtils } from "../lookup/NotePickerUtils";
-import { SchemaPickerUtils } from "../lookup/SchemaPickerUtils";
-import { IDendronQuickInputButton } from "./ButtonTypes";
-import { CREATE_NEW_NOTE_DETAIL, CREATE_NEW_SCHEMA_DETAIL } from "./constants";
-import {
-  ILookupProviderOptsV3,
-  ILookupProviderV3,
-  NoteLookupProviderSuccessResp,
-  OnAcceptHook,
-  OnUpdatePickerItemsOpts,
-  ProvideOpts,
-  SchemaLookupProviderSuccessResp,
-} from "./LookupProviderV3Interface";
-import { transformQueryString } from "./queryStringTransformer";
-import { DendronQuickPickerV2, DendronQuickPickState } from "./types";
-import { OldNewLocation, PickerUtilsV2 } from "./utils";
 
```

Comment:
Any reason to leave these two function here? I think the file name is a bit confusing as well since now it looks like this file only contains some utils.

Reply:
Moved these to the `utils.ts` file in the corresponding lookup folder